configuration/configuration.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // String returns the current configuration as a string
-func (c *ConfigurationData) String() string {
+func (c *Registry) String() string {
```

Comment:
Please use `Data`, as it's already done in `Tenant`, so we'll be more coherent between the repos

Reply:
Coherency is one thing but we embed `viper.Viper` in the `ConfigurationData`. And `viper.Viper` is documented with `Viper is a prioritized configuration registry.`. That's why I would like to keep it as `Registry`.

Reply:
ok, then